src/main/java/com/cs56fitnessapp/models/Day.java

```diff
@@ -1,44 +1,111 @@
 package com.cs56fitnessapp.models;
 
 /**
- * @author Ksenia Koldaeva
- * Created: 9/17/17
- * Last Updated: 10/1/17
+ * @author Saori Shigehisa
+ * Created: 10/6/17
+ * Last Updated: 10/14/17
  */
+import com.cs56fitnessapp.models.workout.Workout;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Date;
+
 
 public class Day implements ActivityFacts, DietFacts {
 
+    /** date */
+    private LocalDate date;
+
+    /** User */
+    private User user;
+
+    /** calorieGoal */
+    private double calorieGoal;
+
```

Comment:
Remove the data field completely:
`private double calorieGoal`
 - as per recent UPD in the UML